multisense_bringup/remote_head.launch

```diff
@@ -44,7 +44,7 @@
       <param name="sensor_ip" value="$(arg ip_address)" />
       <param name="sensor_mtu" value="$(arg mtu)" />
       <param name="tf_prefix"  value="$(arg vpb_tf_prefix)" />
-      <param name="head_id"  value="0" />
+      <param name="head_id"  value="-1" />
```

Comment:
Why are we using a invalid default? Shouldn't we just use 0?

Reply:
The -1 default makes this connect with non-remote head cameras and the VPB. 0 will be remote head ID 0